lib/faker/bot/commands/list.rb

```diff
@@ -11,6 +11,18 @@ def execute(output: $stdout)
           result = Reflectors::List.call(options)
           render(result, output)
         end
+
+        private
+
+        def render(result, output)
+          return not_found(output) if result.empty?
+
+          super(result, output)
+        end
+
+        def not_found(output)
+          output.puts "\nSorry, that class doesn't exist 😢", "\n", "Try something like `Faker::Beer` or `Beer`.", "\n"
```

Comment:
Nice one. Tiny linter suggestion to ease legibility
```suggestion
          output.puts "\nSorry, that class doesn't exist 😢", "\n",
                      'Try something like `Faker::Beer` or `Beer`.', "\n"
```